purchase_request_substate/views/purchase_request_views.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2021 Ecosoft License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_purchase_request_form" model="ir.ui.view">
+        <field name="model">purchase.request</field>
+        <field name="inherit_id" ref="purchase_request.view_purchase_request_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//header/field[@name='state']" position="before">
+                <field
+                    name="substate_id"
+                    widget="statusbar"
+                    options="{'clickable': '1',}"
+                />
+            </xpath>
+        </field>
```

Comment:
```suggestion
            <xpath expr="//header" position="after">
                <header name="substate">
                    <field
                        name="substate_id"
                        widget="statusbar"
                        options="{'clickable': '1',}"
                        domain="[('model', '=', 'purchase.request'),('target_state_value_id.target_state_value', '=', state)]"
                    />
                </header>
            </xpath>
```

I did'nt see the substate in the view.
![image](https://github.com/user-attachments/assets/408d393a-5de7-4145-8d9a-b545f4495a52)